index.bs

```diff
@@ -1387,6 +1390,10 @@ limits the size of filtering IDs within an [=aggregatable attribution report=].
 that controls the allowable values of max bytes. Its value is [=the inclusive
 range|the range=] 1 to 8, inclusive.
 
+<dfn>Default max aggregatable reports per source</dfn> is a positive integer that controls the default
```

Comment:
For clarity, we should also ensure that it is less than or equal to the max settable value.

Reply:
Yes, ensure how?

Reply:
Add a constraint like `It must be less than or equal to [=max aggregatable reports per source=][0].`

Reply:
Done.